homeassistant/components/kmtronic/__init__.py

```diff
@@ -79,6 +79,19 @@ async def async_update_data():
     return True
 
 
+async def async_migrate_entry(
```

Comment:
It doesn't look like we need to change entry version for this migration. Just check in `async_setup_entry` if the reverse option is not set, and update the entry with a default if needed.

We want to avoid entry version migrations since it makes rollbacks impossible or harder.